src/PythonQt.cpp

```diff
@@ -1713,6 +1713,20 @@ void PythonQt::overwriteSysPath(const QStringList& paths)
 void PythonQt::setModuleImportPath(PyObject* module, const QStringList& paths)
 {
   PyModule_AddObject_DECREF(module, "__path__", PythonQtConv::QStringListToPyList(paths));
+#if PY_MAJOR_VERSION >= 3 || (PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION >= 4)
+  // Since Python 3.4 a module has a __spec__ member of type ModuleSpec which
```

Comment:
I think this check should be `#if PY_MAJOR_VERSION > 3 || (PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION >= 4)`, otherwise it would be same as `#if PY_MAJOR_VERSION >= 3`.